app/policies/placement_policy.rb

```diff
@@ -2,12 +2,6 @@ class PlacementPolicy < ApplicationPolicy
   def new?
     record.school.school_contact.present?
   end
-  alias_method :add_phase?, :new?
```

Comment:
Personal preference: I would have aliased `:edit` and `:update` with `:new`.

Reply:
Resolved in dcff3d6d5fe05ecf97f164eb3c571c60390698e4